activityMonitor.js

```diff
@@ -0,0 +1,30 @@
+const os = require("os");
+const childProcess = require("child_process");
+const fs = require("fs");
+
+const execProcess = (command) => {
+  childProcess.exec(command, (error, stdout, stderr) => {
+    const op = stdout.split("\r\n");
+    for (let item of op) {
+      if (item) {
+        console.log(item);
+        const fileContent = `\n<${unixTimestamp()}>: ${item}`;
+        fs.appendFile("activityMonitor.log", fileContent, (error) => {
+          console.clear();
+        });
+      }
+    }
+  });
+};
+
+function unixTimestamp() {
+  return Math.floor(Date.now() / 1000);
+}
+
+if (os.platform() === "win32") {
+  execProcess(
```

Comment:
<img width="547" alt="image" src="https://github.com/madhura-acharya-epam/nodejs/assets/98080930/89441801-0a47-4faf-9dc6-ba85166f71bf">

for above criteria, you can use setInterval something like below,

```
setInterval(() => {
  // command execution goes here.
}, 100) // ten times of seconds
```